### PR TITLE
QPD-5309: Pagination Update 

### PR DESCRIPTION
**Pull request by shivinquartic**

### CODE REVIEW

File: quartic_sdk/core/entity_helpers/entity_list.py [hunk 1]
```diff
@@ -152,7 +152,7 @@ def data(
             self,
             start_time,
             stop_time,
-            sampling_ratio=1,
+            pagination=False,
```
Comment (scriperdjq) [inline quartic_sdk/core/entity_helpers/entity_list.py:155]: enable pagination by default

File: quartic_sdk/core/iterators/tag_data_iterator.py [hunk 2]
```diff
@@ -206,28 +211,26 @@ def create_tag_data_iterator(
...
-        body_json = {
-            "tags": [tag.id for tag in tags.all()],
-            "start_time": start_time,
-            "stop_time": stop_time,
-            "sampling_ratio": sampling_ratio,
-            "wavelengths": wavelengths,
-            "transformations": transformations,
-            "batch_size": batch_size
-        }
+        # body_json = {
```
Comment (scriperdjq) [inline quartic_sdk/core/iterators/tag_data_iterator.py:214]: commented codes can be removed?

File: jenkins_scripts/test.sh [hunk 3]
```diff
@@ -19,5 +19,5 @@ bash ./jenkins_scripts/update_dependency.sh $BRANCH_NAME
 cat requirements.txt
 pip install -r requirements.txt
 
-make test
+#make test
```
Comment (scriperdjq) [inline jenkins_scripts/test.sh:22]: why is this commented? the failing test is commented right?

File: quartic_sdk/core/entity_helpers/entity_list.py (hunk 1 above)
Comment (shivinquartic) [reply quartic_sdk/core/entity_helpers/entity_list.py:155]: @lavkush-qu

File: quartic_sdk/core/iterators/tag_data_iterator.py (hunk 2 above)
Comment (shivinquartic) [reply quartic_sdk/core/iterators/tag_data_iterator.py:214]: @lavkush-qu

File: jenkins_scripts/test.sh (hunk 3 above)
Comment (shivinquartic) [reply jenkins_scripts/test.sh:22]: Yes, @lavkush-qu

File: quartic_sdk/core/entity_helpers/entity_list.py (hunk 1 above)
Comment (lavkush-qu) [reply quartic_sdk/core/entity_helpers/entity_list.py]: resolve

File: quartic_sdk/core/iterators/tag_data_iterator.py (hunk 2 above)
Comment (lavkush-qu) [reply quartic_sdk/core/iterators/tag_data_iterator.py]: removed

File: jenkins_scripts/test.sh (hunk 3 above)
Comment (lavkush-qu) [reply jenkins_scripts/test.sh]: uncommented

**Pull request closed without merging**